README.md

```diff
@@ -14,575 +14,6 @@
 # limitations under the License.
 -->
 
-[![Join the chat at https://gitter.im/thelastpickle/cassandra-medusa](https://badges.gitter.im/thelastpickle/cassandra-medusa.svg)](https://gitter.im/thelastpickle/cassandra-medusa?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+For any support questions or discussion you'd like to start or contribute to, join the channel #cassandra-medusa on ASF slack at [http://s.apache.org/slack-invite](http://s.apache.org/slack-invite)
```

Comment:
I'd go with:

For user questions and general/dev discussions, please join the #cassandra-medusa channel on the ASF slack at [http://s.apache.org/slack-invite](http://s.apache.org/slack-invite)